### DISCUSSION TITLE
[Request] iVentoy LXC Container

### DISCUSSION THREAD

**Discussion by Uzurka** [tteck/Proxmox#2981] [Ideas]
Hello !
I found your git & websites today and i'm kinda amazed by the work done here.
In order to free up some ressources on my Proxmox Hypervisor, i'll replace some of my services with LXC containers using your script.
At the moment, i have a VM used for network containing : 
- Wireguard
- Adguardhome
- iVentoy

The 2 first are already done, so i'm asking for the 3rd :D

**Comment by asylumexp** [comment]
https://github.com/tteck/Proxmox/pull/2790

it was added but then deleted

**Comment by tteck** [comment]
Ready for testing
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/iventoy.sh)"
```
~This will not be added to the website until thorough testing by users.~ Please post results here.

**Comment by Uzurka** [comment]
On my way o/

**Comment by Uzurka** [comment]
Haven'd added ISOs and tried to boot up yet, but installation worked like a charm

**Comment by Uzurka** [comment]
Booted fine. This is working (test done in SSH, not in proxmox web console )

**Comment by systemcrash** [comment]
Can also confirm:
```
    _ _    __           __             
   (_) |  / /__  ____  / /_____  __  __
  / /| | / / _ \/ __ \/ __/ __ \/ / / /
 / / | |/ /  __/ / / / /_/ /_/ / /_/ / 
/_/  |___/\___/_/ /_/\__/\____/\__, /  
                              /____/
Using Default Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 0
Using Root Password: Automatic Login
Using Container ID: 102
Using Hostname: iventoy
Using Disk Size: 2GB
Allocated Cores 1
Allocated Ram 512
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway IP Address: Default
Using Apt-Cacher IP Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a iVentoy LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-zfs for Container Storage.
 ✓ Updated LXC Template List
 ✓ Downloaded LXC Template
 ✓ LXC Container 102 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.250.208 
 ✓ IPv4 Internet Connected
 ✗ IPv6 Internet Not Connected
 ✓ DNS Resolved github.com to 140.82.121.3
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Installed iVentoy
 ✓ Created Service
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

iVentoy should be reachable by going to the following URL.
         http://192.168.250.208:26000/ 
```

Only problem was failing to detect IPv6 which was statically assigned, but generally available on the PVE host bridge.

**Reply by mspearey** [reply]
I've had similar issues with IPv6 on existing scripts, as they use the same function. Still seems to get an IPv6 after it's started.

**Comment by epogo75** [comment]
Hello. I've just installed iVentoy in Proxmox. But where is the "iso" folder in which i must copy the iso files?

**Reply by Uzurka** [reply]
/opt/iVentoy/iso.
My advice : Mount a NFS/CIFS share to this path, or use a large (2Gb default won't be enough, depends on the ISOs you want to use) image for the LXC container

**Reply by MitchellHish** [reply]
I spent far longer than I care to admit stuck trying to find this location. Since it is different than the IVenoty documentation, I might suggest mentioning this on the helper-script site.

That said, this is working great for me, and I wanted to say thank you

**Reply by hereisderek** [reply]
> I spent far longer than I care to admit stuck trying to find this location. Since it is different than the IVenoty documentation, I might suggest mentioning this on the helper-script site.
> 
> That said, this is working great for me, and I wanted to say thank you

yeah this would greatly helped me too

**Reply by ACNSolutions** [reply]
> /opt/iVentoy/iso. My advice : Mount a NFS/CIFS share to this path, or use a large (2Gb default won't be enough, depends on the ISOs you want to use) image for the LXC container

Could you tell me how I do this to use the iso storage in proxmox at /var/lib/vz/template/iso. Sorry, very new to linuxand trying to take it all in!

**Reply by cafec0ffee** [reply]
> > /opt/iVentoy/iso. My advice : Mount a NFS/CIFS share to this path, or use a large (2Gb default won't be enough, depends on the ISOs you want to use) image for the LXC container
> 
> Could you tell me how I do this to use the iso storage in proxmox at /var/lib/vz/template/iso. Sorry, very new to linuxand trying to take it all in!

on proxmox host:
pct set **CTID** -mp0 /var/lib/vz/template/iso,mp=/opt/iventoy/iso/

https://pve.proxmox.com/wiki/Linux_Container#_bind_mount_points

**Reply by ACNSolutions** [reply]
Thanks. I'm learning and I have now successfully set up a mount point to an external nas device containing iso files on a shared folder.
Everything works but when I restart the iventoy container I have to go to the web page to refresh the Image files and then click the big green start button. Do you think this happens because the mount point hasn't started up in time before the iventoy service starts?

**Reply by ACNSolutions** [reply]
I think I have answered my own question as I restarted the service using systemctl restart iventoy.service and all works straight away.
I guess my next question is how to delay the iventoy service starting?  I found this command:

[Service]
ExecStartPre=/bin/sleep 30

So once I have found the service file I will try it out.

**Reply by ACNSolutions** [reply]
Solved! Added sleep 10 to iventoy.sh file.

**Comment by yodaphone** [comment]
Ran this script but unable to start iventoy
when i tried to start it manually, i'm getting this error

```
root@iventoy:~# /opt/iventoy/iventoy.sh status
iventoy is not running.
root@iventoy:~# /opt/iventoy/iventoy.sh start 
env: '/root/lib/iventoy': No such file or directory
iventoy start FAILED
```

**Reply by tteck** [reply]
That's because the Environments are defined in the service file.
https://github.com/tteck/Proxmox/blob/9dac75a48b936680e2d6d959ce7e8c252d53953c/install/iventoy-install.sh#L33-L47

Start it by using `systemctl start iventoy.service`
Check status: `systemctl status iventoy.service`

**Reply by yodaphone** [reply]
still no luck 

i created a new CT with defaults. still no luck

the service starts and shows as running

```
root@iventoy:~# systemctl status iventoy.service
* iventoy.service - iVentoy PXE Booter
     Loaded: loaded (/etc/systemd/system/iventoy.service; enabled; preset: enabled)
     Active: inactive (dead) since Thu 2024-05-16 17:33:27 EDT; 1s ago
       Docs: https://www.iventoy.com
    Process: 4693 ExecStart=sh ./iventoy.sh -R start (code=exited, status=0/SUCCESS)
        CPU: 20ms
```

But the log file has an error

```
2024/05/16 17:33:27.171 [PXE]  ==========================================================
2024/05/16 17:33:27.172 [PXE]  ##########################################################
2024/05/16 17:33:27.172 [PXE]  ##################### IVENTOY RUN ########################
2024/05/16 17:33:27.172 [PXE]  ##########################################################
2024/05/16 17:33:27.172 [PXE]  ==========================================================
2024/05/16 17:33:27.173 [UTIL] Failed to open file <./data/sys/class/dmi/id/product_uuid> errno:13
2024/05/16 17:33:27.173 [UTIL] read sys directory failed 1, errno:13
2024/05/16 17:33:27.173 [PXE]  ### iVentoy start failed. ###
```

**Reply by tteck** [reply]
Are you sure you're creating the container privileged? It must be privileged.

**Reply by yodaphone** [reply]
Darn it... thanks

But i assumed that the Automatic mode will create a privileged container. it didnt for me

anyway thanks

**Reply by tteck** [reply]
If you select "Yes" to "Use Default Settings?" it will in fact create the container privileged.

**Comment by VTFLAB** [comment]
I attempted to mount an NFS directory and use it by mounting it to the ISO directory, but it did not work correctly. After mounting, when I navigated to "Image Management" in the WebUI and performed a refresh, the screen appeared as shown in the attached screenshot.

![Screenshot from 2024-05-17 01-07-09](https://github.com/tteck/Proxmox/assets/10072114/e7fa25de-2ef1-4127-a6ec-65af0c8ad851)

The logs do not provide detailed information and contain only the following entries:
```bash
May 17 01:11:39 iventoy systemd[1]: iventoy.service: Scheduled restart job, restart counter is at 6.
May 17 01:11:39 iventoy systemd[1]: Stopped iventoy.service - iVentoy PXE Booter.
May 17 01:11:39 iventoy systemd[1]: iventoy.service: Start request repeated too quickly.
May 17 01:11:39 iventoy systemd[1]: iventoy.service: Failed with result 'signal'.
May 17 01:11:39 iventoy systemd[1]: Failed to start iventoy.service - iVentoy PXE Booter.
```

*The NFS directory I am attempting to use has been confirmed to work when mounted and used by other LXC containers and VMs.

**Reply by Uzurka** [reply]
Check logs (/opt/iventoy/log/log.txt)
Which args did you past for the NFS mount ? 
Probably a permissions issue

**Reply by VTFLAB** [reply]
@Uzurka 
Thank you for providing the path to the logs. It seems the issue is not related to permissions. Apparently, the service stops when it encounters unsupported ISO files. When I copied the same ISO files to a local directory, the service stopped in the same way. I will organize the ISO files. Thank you for your advice.

**Comment by knario** [comment]
Good afternoon, thank you very much for the iventoy script. I wanted to ask if this script has a web UI interface, as I cannot access it. Thank you very much

**Reply by tteck** [reply]
```
iVentoy should be reachable by going to the following URL.
         http://192.168.250.208:26000/ 
```

Replace IP with yours

**Reply by knario** [reply]
Thank you very much tteck ,that's what I meant with the question. I have put everything with the fixed ip assigned to the lxc and the interface does not work. I have done it in two ways, putting dhcp and fixed ip, that's why I was asking.

**Reply by tteck** [reply]
Create a new LXC using Default settings to see if it works.

**Reply by knario** [reply]
Thank you so much tteck !! it worked by creating a lxc with the automatic configuration. how wonderful. Thanks !!!!

**Comment by flashbackpa** [comment]
> Hello ! I found your git & websites today and i'm kinda amazed by the work done here. In order to free up some ressources on my Proxmox Hypervisor, i'll replace some of my services with LXC containers using your script. At the moment, i have a VM used for network containing :
> 
>     * Wireguard
> 
>     * Adguardhome
> 
>     * iVentoy
> 
> 
> The 2 first are already done, so i'm asking for the 3rd :D

Hi @Uzurka , can I ask you how can you use iVentoy with adguard ?
If i'm not wrong, adguard has yet no support for bootfile and nextserver options, so maybe you use it just for dns and not for dhcp ?
Thank you!

**Reply by Uzurka** [reply]
Hey ! 
Multiple things are to consider in my setup : 
I have a netgear Router with FreshTomato installed on it. So DNSmasq handles my DHCP configuration
AdguardHome is my DNS server, i'm not using his DHCP anymore
ANYMORE because, i used to, when i was using my ISP's router.
iVentoy & AdguardHome's DHCP can run together.
One of the best feature in iVentoy is that he also handle a DHCP proxy configured for PXE boot over your LAN. So simply run the script to create a LXC container, create a VM, boot PXE on it, and iVentoy should boot without configuration needed in AdguarHome :)

**Reply by flashbackpa** [reply]
Thank for the infos, you're right, adguard and iventoy dhcp can coexists, so now iventoy is working ! 👍

**Comment by flashbackpa** [comment]
::SELF FIXED PROBLEM::

Hello everybody, hello tteck, thank you so much for your wonderful work.
I installed without problems iventoy, then i added to fstab the line to mount a remote nfs share, but when i restart iventoy this error popups on the webpage:

No IP address detected.
Attention that iVentoy only supports wired network, wireless network is not supported.

Does this happen to anybody ?
Any potential fix ?

Thanks a lot !

P.S. in the boot information page, the select server ip is empty and there's no ip to choose from..
P.S. 2 i deleted and reinstalled iventoy, and without any modifications rebooted it, and there the error appears...
P.S. 3 ctrl F5 solved the problem 👍

**Reply by Manichee** [reply]
I have this issue, but no matter how many times I refresh or crtl-f5 it still says no ip address detected. First boot of the lxc shows the ip and works. Soon as I reboot it gives that error.

*EDIT*
If I do a "systemctl restart iventoy.service" after the lxc boots it works.

**Reply by andreaconfa** [reply]
> I have this issue, but no matter how many times I refresh or crtl-f5 it still says no ip address detected. First boot of the lxc shows the ip and works. Soon as I reboot it gives that error.
> 
> _EDIT_ If I do a "systemctl restart iventoy.service" after the lxc boots it works.

Same issue here. Did you fixed it?

**Reply by Manichee** [reply]
My workaround is to login to the lxc shell and do `systemctl restart iventoy.service` after the lxc boots up.

**Comment by yodaphone** [comment]
This is more of a general comment about iVentoy. Sadly it only support 20 Clients & its close sourced. would love to hear about comparing this to netboot.xyz

https://netboot.xyz/docs/selfhosting

I have it running on docker and works great

If tteck is ok, then i can raise a request

**Reply by Uzurka** [reply]
Netbootxyz requires a DHCP configuration to handle the PXE boot on it, or a DHCP Proxy, which is a bit of a mess to configure
iVentoy is russian, close sourced, but handles ISO boot without any particular requirement, including Windows ISOs

Netbootxyz allows to boot from web-downloaded images, but iirc, doesn't handle windows without making winPE images and a bit of a mess of configuration ^^

**Comment by cr4sh62** [comment]
Work great :)
As @Uzurka says, just have to mount a NFS/CIFS share to the path "/opt/iventoy/iso" if you have a share with your iso in it so you can stay with a LXC with 2GB drive
Doesn't work for me directly by fstab, but ok with a "mount" command

Thanks @tteck and everybody else for that

**Comment by klingon888** [comment]
Installed and works. BUT, the Boot config settings are not kept after rebooting container. Not sure if this applies but I believe the settings are kept in /data/config.dat but this is missing. Anyone else seeing this issue after reboot?

Log file:
2024/05/18 13:17:15.334 [PXE]  PXE cofigure recovery ...
2024/05/18 13:17:15.334 [PXE]  data/config.dat not found, no need for recovery.
2024/05/18 13:17:15.334 [PXE]  ########## iVentoy auto running. #########
2024/05/18 13:17:15.334 [PXE]  No image file detected.
2024/05/18 13:17:48.161 [HTTP] API request: <{"method":"sysinfo"}>
2024/05/18 13:17:48.226 [HTTP] API request: <{"method":"sys_ip_list"}>
2024/05/18 13:17:48.238 [HTTP] API request: <{"method":"get_dhcp_mode"}>
2024/05/18 13:17:56.163 [HTTP] API request: <{"method":"sysinfo"}>
2024/05/18 13:17:56.218 [HTTP] API request: <{"method":"sys_ip_list"}>
2024/05/18 13:17:56.230 [HTTP] API request: <{"method":"get_dhcp_mode"}>
2024/05/18 13:17:58.026 [HTTP] API request: <{"method":"get_config_all"}>
2024/05/18 13:18:04.140 [HTTP] API request: <{"method":"set_boot_resolution","resolution":"1920x1080"}>
2024/05/18 13:18:14.517 [HTTP] API request: <{"method":"set_dhcp_mode","dhcp_mode":1}>
2024/05/18 13:19:18.299 [HTTP] API request: <{"method":"sysinfo"}>
2024/05/18 13:19:18.344 [HTTP] API request: <{"method":"get_config_all"}>
2024/05/18 13:19:19.850 [HTTP] API request: <{"method":"sysinfo"}>
2024/05/18 13:19:19.899 [HTTP] API request: <{"method":"get_config_all"}>

**Reply by tteck** [reply]
I _think_ the script should create both a `data` and `iso` directory in `/opt/iventoy/`. I'll modify the script.

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/commit/0cd751d75c3d6ffe3c2c618a7b6b44b99076b468

**Reply by klingon888** [reply]
I think the issue is not just the missing config.dat. After many combinations of restarts, the config.dat file got created but subsequent changes to the Boot config settings dont even get updated to the file. Looking at the timestamp, it never changed!

**Comment by klingon888** [comment]
Another problem found but could be an issue with Iventoy and not with yr script. Iventoy crashes when I load "linuxmint-21.3-xfce-64bit.iso" to the iso directory. It refuses to start using "systemctl start iventoy.service". Once I removed this file, it starts normal. This ISO is listed on its "Tested ISO" list, although version linuxmint-21.1. Strange!

root@iventoy:/opt/iventoy/iso# systemctl status iventoy.service
x iventoy.service - iVentoy PXE Booter
     Loaded: loaded (/etc/systemd/system/iventoy.service; enabled; preset: enabled)
     Active: failed (Result: signal) since Sat 2024-05-18 13:31:52 EDT; 7s ago
   Duration: 517ms
       Docs: https://www.iventoy.com
    Process: 584 ExecStart=sh ./iventoy.sh -R start (code=exited, status=0/SUCCESS)
   Main PID: 592 (code=killed, signal=SEGV)
        CPU: 531ms

May 18 13:31:52 iventoy systemd[1]: iventoy.service: Scheduled restart job, restart counter is at 5.
May 18 13:31:52 iventoy systemd[1]: Stopped iventoy.service - iVentoy PXE Booter.
May 18 13:31:52 iventoy systemd[1]: iventoy.service: Start request repeated too quickly.
May 18 13:31:52 iventoy systemd[1]: iventoy.service: Failed with result 'signal'.
May 18 13:31:52 iventoy systemd[1]: Failed to start iventoy.service - iVentoy PXE Booter.

Logfile:
2024/05/18 13:31:52.111 [PXE]  Scan iso directory for all image files ...
2024/05/18 13:31:52.111 [PXE]  Add ISO <iso/linuxmint-21.3-xfce-64bit.iso> size:3033710592 type:0
2024/05/18 13:31:52.111 [PXE]  [1] Phase1 parse image <iso/linuxmint-21.3-xfce-64bit.iso> <ISO> 0 starting...
2024/05/18 13:31:52.121 [PXE]  [1] img <iso/linuxmint-21.3-xfce-64bit.iso> <ISO> OS type:Linux
2024/05/18 13:31:52.121 [PXE]  [1] Check linux arch type for <iso/linuxmint-21.3-xfce-64bit.iso>

**Reply by gladiatormax** [reply]
I just got the same error, very strange.

**Comment by styxadmin** [comment]
Works great, except for the fact the the resolution is messing up the image (its tearing or smearing)

When using on multiple clients one has to constantly play with the screen resolution

**Comment by cafec0ffee** [comment]
run as unprivileged lxc
https://github.com/ventoy/PXE/issues/53#issuecomment-2129841449

**Reply by styxadmin** [reply]
> run as unprivileged lxc [ventoy/PXE#53 (comment)](https://github.com/ventoy/PXE/issues/53#issuecomment-2129841449)

I tried that, but the commands mentioned by @cafec0ffee do not work.

Launching the web interface does not bring up the expected iVentoy WebUI

**Reply by cafec0ffee** [reply]
> > run as unprivileged lxc [ventoy/PXE#53 (comment)](https://github.com/ventoy/PXE/issues/53#issuecomment-2129841449)
> 
> I tried that, but the commands mentioned by @cafec0ffee do not work.
> 
> Launching the web interface does not bring up the expected iVentoy WebUI

show me what your command shows in the unprivileged container
`ls -l /sys/class/dmi/id/product_uuid`

delete iventoy/log/log.txt
run  `sh ./iventoy.sh -R start`
and show what log.txt says



my steps:
create unprivileged container (I used debian-12-standard_12.2-1_amd64.tar.zst)
```
#cat /etc/pve/lxc/123.conf
arch: amd64
cores: 1
features: nesting=1
hostname: iventoy
memory: 512
net0: name=eth0,bridge=vmbr0,firewall=1,gw=192.168.1.1,hwaddr=BC:24:11:5D:38:9A,ip=192.168.1.123/24,type=veth
ostype: debian
rootfs: HDD:subvol-123-disk-0,size=8G
swap: 512
unprivileged: 1
```
add permission on proxmox host
```
#chmod 444 /sys/class/dmi/id/product_uuid
#ls -l /sys/class/dmi/id/product_uuid
-r--r--r-- 1 root root 4096 May 24 20:27 /sys/class/dmi/id/product_uuid
#crontab -e
@reboot /usr/bin/chmod 444 /sys/class/dmi/id/product_uuid
```

in container:
```
#ls -l /sys/class/dmi/id/product_uuid
-r--r--r-- 1 nobody nogroup 4096 May 24 17:05 /sys/class/dmi/id/product_uuid

#cd /opt && wget https://github.com/ventoy/PXE/releases/download/v1.0.20/iventoy-1.0.20-linux-free.tar.gz
#tar zxf iventoy-1.0.20-linux-free.tar.gz && rm $_
#mv iventoy-1.0.20 iventoy

#cat /etc/systemd/system/iventoy.service
[Unit]
Description=iVentoy PXE Booter
Documentation=https://www.iventoy.com
Wants=network-online.target
[Service]
Type=forking
Environment=IVENTOY_API_ALL=1
Environment=IVENTOY_AUTO_RUN=1
Environment=LIBRARY_PATH=/opt/iventoy/lib/lin64
Environment=LD_LIBRARY_PATH=/opt/iventoy/lib/lin64
ExecStart=sh ./iventoy.sh -R start
WorkingDirectory=/opt/iventoy
Restart=on-failure
[Timer]
OnActiveSec=5sec
AccuracySec=1s
[Install]
WantedBy=multi-user.target

#systemctl --now enable iventoy

#ss -tulpn
Netid    State     Recv-Q    Send-Q        Local Address:Port          Peer Address:Port    Process
tcp      LISTEN    0         511                 0.0.0.0:26000              0.0.0.0:*        users:(("iventoy",pid=329,fd=24))

#systemctl status iventoy
* iventoy.service - iVentoy PXE Booter
     Loaded: loaded (/etc/systemd/system/iventoy.service; enabled; preset: enabled)
     Active: active (running) since Sat 2024-05-25 11:25:01 UTC; 12min ago
       Docs: https://www.iventoy.com
    Process: 323 ExecStart=sh ./iventoy.sh -R start (code=exited, status=0/SUCCESS)
   Main PID: 329 (iventoy)
      Tasks: 12 (limit: 154324)
     Memory: 25.3M
        CPU: 816ms
     CGroup: /system.slice/iventoy.service
             `-329 /opt/iventoy/lib/iventoy

May 25 11:25:01 iventoy systemd[1]: Starting iventoy.service - iVentoy PXE Booter...
May 25 11:25:01 iventoy sh[323]: iventoy start SUCCESS PID=0
May 25 11:25:01 iventoy sh[323]: Please open your browser and visit http://127.0.0.1:26000 or http://x.x.x.x:26000 (x.x.x.x is any valid IP address)
May 25 11:25:01 iventoy systemd[1]: Started iventoy.service - iVentoy PXE Booter.

```



it works great for me

**Reply by cafec0ffee** [reply]
lol the simplest option add this **lxc.hook.pre-start: sh -c "chmod 444 /sys/class/dmi/id/product_uuid"** into configuration

```
#cat /etc/pve/lxc/123.conf
arch: amd64
cores: 1
features: nesting=1
hostname: iventoy
memory: 512
net0: name=eth0,bridge=vmbr0,firewall=1,gw=192.168.1.1,hwaddr=BC:24:11:F6:85:9B,ip=192.168.1.123/24,type=veth
ostype: debian
rootfs: HDD:subvol-123-disk-0,size=8G
swap: 512
unprivileged: 1
lxc.hook.pre-start: sh -c "chmod 444 /sys/class/dmi/id/product_uuid"
lxc.hook.post-stop: sh -c "chmod 400 /sys/class/dmi/id/product_uuid"
```

**Reply by styxadmin** [reply]
> run as unprivileged lxc [ventoy/PXE#53 (comment)](https://github.com/ventoy/PXE/issues/53#issuecomment-2129841449)

I think I got my wires crossed. I mistakenly assumed (blame github notifications being weird) that your commend was in response to my issue of the display resolution issue with iVentoy.

Please ignore my comments

**Reply by snickerweb** [reply]
Hello, thank you TTeck for including iVentoy. Very cool tool.

Maybe it would be useful to provide a hint as to the location of the ISO. 2GB is not enough for ISOs ;)
Even better, you ask whether you want to automatically map to the Proxmox ISO directory.
In my case with:

pct set 211 -mp0 /var/lib/vz/template/iso,mp=/opt/iventoy/iso

**Reply by imuix** [reply]
pct set 211 -mp0 /var/lib/vz/template/iso,mp=/opt/iventoy/iso

this work for me well 
thank you,

**Comment by ACNSolutions** [comment]
What is the password for ssh login please to iventoy container

**Reply by tteck** [reply]
Did you use default settings, or did you opt for advanced settings such as setting a root password and enabling SSH?

**Reply by ACNSolutions** [reply]
I used deafault settings

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/385
